docs/deployments/deploying-behind-a-proxy.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Deploying a Clerk app behind a proxy
+description: Learn how to deploy a Clerk app to a web server using proxies and reverse proxies like nginx or Caddy, or using Docker
+---
+
+When deploying a Clerk app beind a proxy, you may need to proxy two headers for Clerk to work correctly. The two headers that you must forward are:
```

Comment:
the first sentence says they "may need to" proxy the two headers. the second sentence says they "must" forward the headers.

is it required, or is it not required?

Reply:
Lets use more assured its required tense, I think its safe to assume it needed in nearly if not all network configs.